velox/dwio/common/BitPackDecoder.h

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include "velox/common/base/BitUtil.h"
+#include "velox/common/base/Exceptions.h"
+#include "velox/vector/TypeAliases.h"
+
+#include <folly/Range.h>
+
+namespace facebook::velox::dwio::common {
+
+using RowSet = folly::Range<const facebook::velox::vector_size_t*>;
```

Comment:
This probably should be in `Common.h` or even somewhere in `common/base`

Reply:
@Yuhta Jimmy, if we move it to Common.h, it would make it depend on folly. Is this ok? I can move it once you confirm.

Reply:
OK we can leave it here for now.  It has been redefined in multiple places, we should unify it at some point.